airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetrics.py

```diff
@@ -0,0 +1,24 @@
+# generated by datamodel-codegen:
+#   filename:  ConnectorMetrics.yaml
+
+from __future__ import annotations
+
+from typing import Any, Optional
+
+from pydantic import BaseModel, Extra
+from typing_extensions import Literal
+
+
+class ConnectorMetrics(BaseModel):
+    all: Optional[Any] = None
+    cloud: Optional[Any] = None
+    oss: Optional[Any] = None
+
+
+class ConnectorMetric(BaseModel):
+    class Config:
+        extra = Extra.allow
+
+    usage: Optional[Literal["none", "low", "medium", "high"]] = None
+    sync_success_rate: Optional[Literal["none", "low", "medium", "high"]] = None
+    connector_version: Optional[str] = None
```

Comment:
❓ when might we have or not have version info for this? does it affect how the usage and sync_success_rate are calculated?

Reply:
especially curious because i see usage of `connector_version: all` in the upstack pr

Reply:
This is more or less a design choice resulting from
1. We want metrics to be in control of the Data team
2. We dont want any metrics added or removed to require a code change in our systems
3. The file which they give us is outside of our controll.

This led me to put in a Type that outlines what we know so far. But
1.  makes no guarantees that we will always have a value for these fields
2. allows for additional properties